src/components/User/User.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person,
+  people: Person[],
+  selectedUser?: Person,
+};
+
+const findRelative = (gender: string, user: Person, users: Person[]) => {
+  if (gender === 'm') {
+    return users.filter(oneUser => oneUser.sex === 'm')
+      .find(oneUser => oneUser.name === user.fatherName);
+  }
+
+  if (gender === 'f') {
+    return users.filter(oneUser => oneUser.sex === 'f')
+      .find(oneUser => oneUser.name === user.motherName);
+  }
+
+  return undefined;
+};
+
+export const User: React.FC<Props> = ({ person, people, selectedUser }) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    slug,
+  } = person;
+
+  const findMother = findRelative('f', person, people);
+  const findFather = findRelative('m', person, people);
+
+  return (
+    <tr
+      key={slug}
+      className={cn({ 'has-background-warning': slug === selectedUser?.slug })}
+      data-cy="person"
+    >
+      <td>
+        <Link
```

Comment:
According to the task, you have to create <PersonLink person={person} />.
Looks much neater this way.
Looky here:
![image](https://github.com/mate-academy/react_people-table-basics/assets/110335429/63f69fbc-bc5f-4d56-ad95-08d131bc8374)